nifi-commons/nifi-properties/src/main/java/org/apache/nifi/util/NiFiProperties.java

```diff
@@ -71,6 +71,7 @@
     public static final String ADMINISTRATIVE_YIELD_DURATION = "nifi.administrative.yield.duration";
     public static final String PERSISTENT_STATE_DIRECTORY = "nifi.persistent.state.directory";
     public static final String BORED_YIELD_DURATION = "nifi.bored.yield.duration";
+    public static final String PROCESSOR_START_TIMEOUT = "nifi.processor.start.timeout";
```

Comment:
This needs to be added to the nifi.properties file, with a default value added to the assembly's pom.xml, I believe. Also, I believe as-is, it expects a number of milliseconds. Would like to see that changed to using the standard "Time format" that we use in nifi - e.g. "5 secs" instead of "5000"